src/integrations/prefect-azure/pyproject.toml

```diff
@@ -77,6 +77,7 @@ version_file = "prefect_azure/_version.py"
 root = "../../.."
 tag_regex = "^prefect-azure-(?P<version>\\d+\\.\\d+\\.\\d+)$"
 fallback_version = "0.0.0"
+git_describe_command = 'git describe --dirty --tags --long --match "prefect-azure-*[0-9]*"'
```

Comment:
I needed to update this to ensure we are using the write tag format when trying to determine a version.